lib/constants.js

```diff
@@ -27,7 +33,7 @@ export const PRIZE_POOL_TYPE = Object.freeze({
   yield: 'yield'
 })
 
-export const SUPPORTED_NETWORKS = [1, 4, 42, 77, 100, 137, 31337, 1234]
+export const SUPPORTED_NETWORKS = [1, 4, 42, 56, 77, 97, 100, 137, 31337, 1234]
```

Comment:
99 `POA Network Core` was added to the ref app in https://github.com/pooltogether/pooltogether-reference-pool-ui/pull/57, do we want that?

Reply:
Y'know I thought we did, but couldn't find deployments from the contracts team:

```
ls node_modules/@pooltogether/pooltogether-contracts/deployments/                                                 
bsc/         bscTestnet/  kovan/       localhost/   mainnet/     matic/       mumbai/      poaSokol/    rinkeby/     xdai/     
```